### ISSUE TITLE
[LINT_BUG]: 

### ISSUE THREAD

**Issue by TymekDev** [Appsilon/box.linters#153]
### box.linters version

0.10.5

### Sample source code to lint

Make sure to have `{here}` **not** installed.

**`foo.R`**
```r
box::use(
  ./bar[asdf]
)
```

**`bar.R`**
```r
box::use(
  here[here],
)

#' @export
asdf <- here
```

### Lint command used

```r
lintr::lint(
  "./foo.R",
  linters = box.linters::box_mod_fun_exists_linter()
)
```

### Lint result

Linters fails to run with the following error:
```
Error: Linter 'box.linters::box_mod_fun_exists_linter' failed in <snip>/foo.R: there is no package called ‘here’
```

### Expected result

I would expect an error lint and/or a graceful handling of the missing package.

**Comment by TymekDev** [comment]
It's caused by loading the module:
https://github.com/Appsilon/box.linters/blob/27df08a0bed9e5576130a344b29c963d8760be2e/R/get_box_module_exports.R#L32
